### DISCUSSION TITLE
Save Layout of Nodes

### DISCUSSION THREAD

**Discussion by jkanel** [open-source-labs/Svelvet#336] [Q&A] [answered]
The use case is that I want to save to a database the layout of `<Nodes>`.  Is there a recommended approach to achieve this?

Related question: is it possible to bind a Svelvet element to a variable that could then be inspected in code to read node positions, etc?

Thanks!

**Comment by briangregoryholmes** [comment, marked as answer]
A native method for save and restore will likely be part of the next major release.

In v7.0.47, you can bind to the position prop on a Node. Except for the initial setting of the position, the binding is currently read only.

```html
<script lang="ts">
	import { Svelvet, Node } from 'svelvet';
	let position = { x: 300, y: 300 };
        $: console.log(position)
</script>

<body>
	<Svelvet >
		<Node bind:position />
	</Svelvet>
</body>
```
